packages/picasso/src/Calendar/Calendar.tsx

```diff
@@ -81,6 +81,8 @@ export const Calendar = forwardRef<HTMLDivElement, Props>(function Calendar(
     disabledIntervals,
     renderDay,
     weekStartsOn,
+    renderRoot = rootProps => <CalendarContainer {...rootProps} />,
+    renderMonthHeader = CalendarMonthHeader,
```

Comment:
nit: For consistency?
```suggestion
    renderRoot = CalendarContainer,
    renderMonthHeader = CalendarMonthHeader,
```

Reply:
in this case, I would have an error